doc/man3/SSL_CONF_cmd.pod

```diff
@@ -532,6 +532,9 @@ B<KTLS>: Enables kernel TLS if support has been compiled in, and it is supported
 by the negotiated ciphersuites and extensions. Equivalent to
 B<SSL_OP_ENABLE_KTLS>.
 
+B<StrictCertCheck>: Enable strict certificate checking. Equivalent to
+setting B<SSL_CERT_FLAG_TLS_STRICT> with SSL_CTX_set_cert_flags().
+
```

Comment:
Just noting that this flag is the equivalent to `-strict` on the s_client/s_server command line, but none of the command-line options reference config options (just the flags).